### ISSUE TITLE
MazSelect : Adding a button to add more 

### ISSUE THREAD

**Issue by Timothy16** [LouisMazel/maz-ui#542]
For MazSelect component, can a button be added to add data if the data searched for cannot be found on the list of data provided in the dropdown.

**Comment by LouisMazel** [comment]
Hi @Timothy16,

I'm not sure what you mean. Do you want a clear button?

**Comment by Timothy16** [comment]
Not clear button. i need a template to display no-options

**Comment by LouisMazel** [comment]
This slot slot is now available in v3.18.0

Check out the doc: https://louismazel.github.io/maz-ui-3/components/maz-select#search-option

Now, you can: 

```html
<MazSelect>
  <template no-results>
    ...
  </template>
</MazSelect>
```
